src/augeas.c

```diff
@@ -2020,6 +2021,39 @@ int aug_escape_name(augeas *aug, const char *in, char **out) {
     return result;
 }
 
+int aug_filter_file(struct augeas *aug, const char *file) {
+    int result = -1;
+
+    api_entry(aug);
+
+    bool found = false;
+
+    list_for_each(module, aug->modules) {
```

Comment:
Oh I see now. If I pass an `xfm` without a `file`, then it will load all the files for that `xfm`. Makes sense…